packages/react/package.json

```diff
@@ -23,7 +23,7 @@
     "keyname": "^0.1.0",
     "prop-types": "^15.6.0",
     "rc-tooltip": "^3.7.2",
-    "react-syntax-highlighter": "^6.1.1",
+    "react-syntax-highlighter": "^13.5.1",
```

Comment:
Updating this is technically a breaking change, [as `react-syntax-highlighter` does not work with Parcel (v1)](https://github.com/parcel-bundler/parcel/issues/3176).

This will prevent us from updating Cauldron in the ["axe Linter" frontend](https://github.com/dequelabs/axe-linter/tree/develop/src/frontend). I don't know if we need to worry about that right now, but it's something we should consider before merging this.

Reply:
/cc @WilcoFiers @michael-siek heads up

Reply:
> Updating this is technically a breaking change

Yes it is, but I figured I didn't have to worry about it since we are still pre v1. If we want to go ahead and tag v1 now (from the dev branch) before we merge this, I'm fine with that. If we do, I will add `BREAKING CHANGE` to the commit message body.

Reply:
No need. The axe Linter folks just said this is fine (on Slack).